src/main/java/org/jalasoft/moi/model/python/PythonCommandBuilder.java

```diff
@@ -54,10 +73,7 @@ private String commandToCompile(Path path) {
      * @param path location of the directory or file.
      * @return execution command.
      */
-    private String commandToRun(Path path) {
-        String fileName = path.getFileName().toString().replace(".py", "");
-        return pythonPath + SPACE +
-               Paths.get(path.getParent() + "/__pycache__/" +
-               fileName + ".cpython-" + version + ".pyc");
+    public String commandToRun(Path path) {
+        return pythonPath + SPACE + path.toString();
```

Comment:
I'm not sure if it's a good idea to leave out ` "/__pycache__/" ` from the path

Reply:
The /__pycache__/ contains the files that was compiled (.pyc), but with python, in order to execute the program, we could run the .py file